src/viewmodels/platform.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <div data-bind="foreach: { data: options, as: 'option'}">
+    <input
+      type="checkbox"
+      data-bind="checked: $parent.checkPlatform, attr: { value: option.name }"
```

Comment:
The value of the checkbox should be option.value instead of option.name as in https://github.com/lrn2prgrm/pyout-frontend/pull/8/commits/8828668bd16c79586faf3ccd6d11a0b50a3a10e6